sensor_msgs/PointCloud2 cloud1
sensor_msgs/PointCloud2 cloud2
---
sensor_msgs/PointCloud2 combined_cloud
geometry_msgs/Pose cloud_center
